dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/metrics/BatchMetricsDispatcherTest.kt

```diff
@@ -608,13 +608,13 @@ internal class BatchMetricsDispatcherTest {
     }
 
     private fun Forge.forgeAGrantedDirName(): String {
-        val separator = if (aBool()) "_" else "-"
+        val separator = "-"
```

Comment:
can be just as `joinToString` argument like `joinToString(separator = "-")`. Same below